x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

```diff
@@ -170,3 +211,27 @@ echo -n "VuaCfGcBCdbkQm-e5aOx:ui2lp2axTNmsyakw9tvNnw" | base64 <1>
 ----
 <1> Use `-n` so that the `echo` command doesn't print the trailing newline
 character
+
+The following example creates an API key with a <<role-restriction, restriction>> to the `search_application_query` workflow,
+which allows to call only <<search-application-search, Search Application Search API>>:
+
+[source,console]
+----
+POST /_security/api_key
+{
+  "name": "my-restricted-api-key",
+  "role_descriptors": {
+    "restricted-role": {
```

Comment:
```suggestion
    "my-restricted-role-descriptor": {
```

Reply:
Just a suggestion. It wasn't clear to me initially that `restricted-role` is user-chosen, rather than a keyword provided by the API.

Reply:
Yes, the `restricted-role` is user-chosen. Thanks for the suggestion. The `my-restricted-role-descriptor` reads better.